### ISSUE TITLE
[Bug]: TPCC 10w 10 terminals test was hung when running for about several minutes on esk with 2*cn

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#8981]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 0f1162f
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

When run  TPCC 10w 10 terminals test on eks with 2*cn, the test was hung after several minutes:

<img width="1243" alt="image" src="https://user-images.githubusercontent.com/47846308/231659850-d71b9b2a-5189-4b88-af7e-1411593e9925.png">
<img width="1402" alt="image" src="https://user-images.githubusercontent.com/47846308/231660305-3fe01aca-e34c-44aa-8c16-758e1a2691d7.png">

[tpcc-hung-cn.txt](https://github.com/matrixorigin/matrixone/files/11218382/tpcc-hung-cn.txt)

and in s3 path, s3://mo-ares/mo-benchmark/etl_202304131127, can get another logs for this issue 

### Steps to Reproduce

```Markdown
1. run a mo cluster on eks with (2*cn, 1*dn, 3*log)
2. load tpcc 10w data 
3. run tpcc 10w 10 terminal test for about 3 minutes 

or 

1. git clone https://github.com/matrixorigin/mo-auto-test.git
2. modify .github/workflows/regression-on-distributed-mode.yml, 
  set JobName.default to tpcc_benchmark_test
  set RunTimes to 10
  
3. make a pr to merge, and the job will automatically run the tpcc 10w 10 terminal test on eks
```

**Comment by aressu1985** [comment]
the last valid commit is 701b6f1

**Comment by ouyuanning** [comment]
happened in pull mode

**Comment by zhangxu19830126** [comment]
16 @ 0x447116 0x4132fb 0x412df8 0x1ed817d 0x1ed82b7 0x1ed8328 0x1ed7d02 0x1f3d5c6 0x1f3e265 0x1f6e849 0x1f6ed46 0x478be1
#	0x1ed817c	github.com/matrixorigin/matrixone/pkg/sql/colexec/dispatch.(*Argument).waitRemoteRegsReady+0x17c	/go/src/github.com/matrixorigin/matrixone/pkg/sql/colexec/dispatch/dispatch.go:124
#	0x1ed82b6	github.com/matrixorigin/matrixone/pkg/sql/colexec/dispatch.sendToAllRemoteFunc+0x2b6			/go/src/github.com/matrixorigin/matrixone/pkg/sql/colexec/dispatch/sendfunc.go:70
#	0x1ed8327	github.com/matrixorigin/matrixone/pkg/sql/colexec/dispatch.sendToAllFunc+0x27				/go/src/github.com/matrixorigin/matrixone/pkg/sql/colexec/dispatch/sendfunc.go:90
#	0x1ed7d01	github.com/matrixorigin/matrixone/pkg/sql/colexec/dispatch.Call+0x161					/go/src/github.com/matrixorigin/matrixone/pkg/sql/colexec/dispatch/dispatch.go:118
#	0x1f3d5c5	github.com/matrixorigin/matrixone/pkg/vm.Run+0x165							/go/src/github.com/matrixorigin/matrixone/pkg/vm/vm.go:61
#	0x1f3e264	github.com/matrixorigin/matrixone/pkg/vm/pipeline.(*Pipeline).MergeRun+0x124				/go/src/github.com/matrixorigin/matrixone/pkg/vm/pipeline/pipeline.go:151
#	0x1f6e848	github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun+0x648				/go/src/github.com/matrixorigin/matrixone/pkg/sql/compile/scope.go:106
#	0x1f6ed45	github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun.func2+0x25				/go/src/github.com/matrixorigin/matrixone/pkg/sql/compile/scope.go:91

**Comment by zhangxu19830126** [comment]
When there is a problem, it catches this stask every time, stops all sql accesses and still exists.

**Comment by zhangxu19830126** [comment]
hung by the compile's channel, next week and computing colleagues to continue the analysis, root cause

**Comment by dongdongyang33** [comment]
same cause with #8996 
<img width="1701" alt="image" src="https://user-images.githubusercontent.com/47596332/232381168-b62a92ab-c1ff-44d5-9d81-25a93a7e9de7.png">

**Comment by aressu1985** [comment]
testing

**Comment by aressu1985** [comment]
fixed:1084951
